test/e2e/yaml/e2e-tools.yaml

```diff
@@ -55,3 +55,25 @@ tools:
         valuefile: values.yaml
         path: helm/${{github-repo-scaffolding-golang.default.outputs.repo}}
         repoURL: ${{github-repo-scaffolding-golang.default.outputs.repoURL}}
+  - name: trello
```

Comment:
Please split the tests for `trello`/`trello-github-integ` into a separate yaml to avoid mixing with GitOps workflow

Reply:
has been split into independent test logic units